### DISCUSSION TITLE
ノートの置き場所

### DISCUSSION THREAD

**Discussion by masahiko-ikau-com** [innovation1005/linear-algebra#13] [General]
いつもお世話になっています。kauです。
ディスカッションを作っていただきましたが、自分で作った勉強用のノートなどはこちらに投稿したほうが良いでしょうか？
それともこっちは会話だけ、という扱いでしょうか？

**Comment by innovation1005** [comment]
今日は込み入っているので明日考えましょう。

 

From: masahiko-ikau-com ***@***.***> 
Sent: Wednesday, June 29, 2022 12:54 PM
To: innovation1005/linear-algebra ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [innovation1005/linear-algebra] ノートの置き場所 (Discussion #13)

 

いつもお世話になっています。kauです。
ディスカッションを作っていただきましたが、自分で作った勉強用のノートなどはこちらに投稿したほうが良いでしょうか？
それともこっちは会話だけ、という扱いでしょうか？

**Comment by innovation1005** [comment]
よくまとまっているようなので、今日もこの資料の説明からお願いできますか？

 

よろしくお願いいたします。

 

 

From: masahiko-ikau-com ***@***.***> 
Sent: Wednesday, June 29, 2022 12:54 PM
To: innovation1005/linear-algebra ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [innovation1005/linear-algebra] ノートの置き場所 (Discussion #13)

 

いつもお世話になっています。kauです。
ディスカッションを作っていただきましたが、自分で作った勉強用のノートなどはこちらに投稿したほうが良いでしょうか？
それともこっちは会話だけ、という扱いでしょうか？

**Reply by masahiko-ikau-com** [reply]
ありがとうございます。承知しました。
